boards/arm/lpcxpresso55s36/Kconfig.defconfig

```diff
@@ -9,3 +9,13 @@ config BOARD
 	default "lpcxpresso55s36"
 
 endif # BOARD_LPCXPRESSO55S36
+
+if DMA_MCUX_LPC
+
+# Memory from the heap pool is used to allocate DMA descriptors for
+# channels that use multiple blocks for a DMA transfer.
+# Adjust HEAP_MEM_POOL_SIZE in case you need more memory.
+config HEAP_MEM_POOL_SIZE
+	default 4096
```

Comment:
this seems strange because it might make the heap pool size smaller than otherwise needed for something else. Can we do: `if DMA_MCUX_LPC && HEAP_MEM_POOL_SIZE < 4096`

It looks like the kconfig language supports this: https://www.kernel.org/doc/html/latest/kbuild/kconfig-language.html#menu-dependencies

Reply:
> What is the heap needed for here?

There are channel specific data structures allocated in the DMA driver. It is allocated based on the number of DMA channels needed by the application via device tree.

Reply:
This is no longer needed, please delete.

Reply:
Thank you, I have removed this.